### PR TITLE
[PLAT-1108] Set text when changes on the pin

### PR DESCRIPTION
**Pull request by MadisonEhlers-Vertex**
## Summary

When the pin changes, set the appropriate text

### CODE REVIEW

File: packages/viewer/src/components/viewer-pin-label/readme.md [hunk 1]
```diff
@@ -36,24 +36,6 @@ Type: `Promise<void>`
 
 
 
-## CSS Custom Properties
-
-| Name                                                  | Description                                                                           |
-| ----------------------------------------------------- | ------------------------------------------------------------------------------------- |
-| `--viewer-annotations-pin-label-background-color`     | A CSS color that specifies the color of the label background                          |
```
Comment (MadisonEhlers-Vertex) [inline packages/viewer/src/components/viewer-pin-label/readme.md:43]: Woops. Will get this back

File: packages/viewer/src/components/viewer-pin-label/viewer-pin-label.tsx [hunk 2]
```diff
@@ -193,6 +198,7 @@ export class VertexPinLabel {
             disabled={!this.focused}
             ref={(ref) => (this.inputEl = ref)}
             value={this.value}
+            autoFocus={false}
```
Comment (jdm717) [inline packages/viewer/src/components/viewer-pin-label/viewer-pin-label.tsx:201]: Is this necessary? I don't believe the default `autoFocus` value is `true`

File: packages/viewer/src/components/viewer-pin-label/viewer-pin-label.tsx [hunk 3]
```diff
@@ -383,15 +397,15 @@ export class VertexPinLabel {
     window.removeEventListener('pointerup', this.handlePointerUp);
   };
 
-  private handleTextBlur = (): void => {
+  private handleTextBlur = async (): Promise<void> => {
```
Comment (jdm717) [inline packages/viewer/src/components/viewer-pin-label/viewer-pin-label.tsx:400]: Any reason this still needs to be async?

**Review by jdm717 — APPROVED** [review]
LGTM, just had a few nits

**Pull request merged**